### PR TITLE
Draft of gas station topic for nextGen docs, update contract interact…

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
…ions

### CODE REVIEW

File: docs/coding-projects/contract-interactions.md [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
 
 The `auth` module manages user authorization, while the `payments` module handles transferring value between two accounts—`Sarah` and `James`.
 
 <img src="/img/contract-interactions-overview.webp" alt="Contract interactions project overview" height="550" width="900"/>
 
+In this tutorial, you'll learn the following:
+
+- Understanding contract interactions in Pact
+- Setting up multiple modules for authorization and payments
+- Testing interactions in the REPL environment
```
Comment (emilypi) [inline docs/coding-projects/contract-interactions.md:27]: love this

File: docs/coding-projects/contract-interactions.md [hunk 2]
```diff
@@ -58,250 +65,203 @@ To get started:
...
+   ```pact
+     (defschema user
+       nickname:string
+       keyset:keyset)
+     
+     (deftable users:{user})
+   ```
 
-Define the schema and table for account management.
+2. Define the create-user function to add a new user to the system, restricted to the operate-admin.
```
Comment (emilypi) [inline docs/coding-projects/contract-interactions.md:112]:
```suggestion
2. Define the create-user function to add a new user to the system, that only the `operate-admin` keyset owners may execute.
```

File: docs/coding-projects/gas-station.md [hunk 3]
```diff
@@ -0,0 +1,654 @@
...
+
+      (implements gas-payer-v1)
+
+   )
+   ```
+ 
+   To deploy the gas station module, you must use an existing namespace such as the `"free"` namespace.
+   For local testing, you can create the `test-gas` module in a custom namespace that you define for the local development network.
+
+   After the `implements` keyword, the `test-gas` module must implement the capabilities, types, and functions as defined in the `gas-payer-v1.pact` file.
```
Comment (emilypi) [inline docs/coding-projects/gas-station.md:144]: "After th eimplements keyword" is  kind of a weird way to say this. I think it means "once an `implements` statement is issued, the implementing module must implement the ...." etc.

File: docs/coding-projects/contract-interactions.md (hunk 1 above)
Comment (rsoeldner) [inline docs/coding-projects/contract-interactions.md:27]: Should we add also:
 - See how you can deploy
?

**Review by rsoeldner — COMMENTED** [review]
nice work @lsgunnlsgunn

File: docs/coding-projects/gas-station.md [hunk 4]
```diff
@@ -0,0 +1,654 @@
...
+
+2. Open and unlock the Chainweaver desktop or web application.
+
+3. Verify that you're connected to **development network (devnet)** from the network list.
+
+1. Verify that your administrative account name with the **k:** prefix exists on the target chain.
+
+1. Verify that your administrative account name is funded with coins on the target chain.
+
+### Load the module using Chainweaver
```
[the remark this answers is not in the document]
Comment (emilypi) [reply docs/coding-projects/gas-station.md:520]: Probably not great, no

**Pull request merged**